src/graphql/graphql-client.ts

```diff
@@ -63,9 +63,30 @@ function createClient(url: string) {
         'X-Correlation-Id': headers['correlationId'],
       },
     });
-    const errorLink = onError((error) =>
-      console.log('Apollo Error:', JSON.stringify(error)),
-    );
+    const errorLink = onError(({operation, graphQLErrors, networkError}) => {
+      let error = '';
+      if (graphQLErrors) {
+        graphQLErrors.forEach(({message, locations, path}) => {
+          error = `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`;
+        });
```

Comment:
This will overwrite the error message with the last one if there are more than one. But in practice they might all be the same one, so it's fine? If not, maybe we could do something like

```suggestion
        graphQLErrors.forEach(({message, locations, path}) => {
          error += `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}\n`;
        });
```

Reply:
Not really sure how the structure looks like. I was expecting just one error, but if it is more than one probably this is safer.